test/controllers/IneligibleControllerSpec.scala

```diff
@@ -359,6 +360,25 @@ class IneligibleControllerSpec extends ItSpec {
         leadingP1 = expectedLeadingP1
       )
       ContentAssertions.commonIneligibilityTextCheck(page, TaxRegime.Vat, Languages.English)
+
+      val expectedTaxDetailsJson = """{ "vrn": "101747001" }"""
+
+      AuditConnectorStub.verifyEventAudited(
+        "DirectDebitInProgress",
+        Json.parse(
+          s"""
+             |{
+             |  "origin": "Bta",
+             |  "taxType": "Vat",
+             |  "taxDetail": $expectedTaxDetailsJson,
+             |  "correlationId": "CorrelationId(8d89a98b-0b26-4ab2-8114-f7c7c81c3059)",
+             |  "authProviderId": "GGCredId(authId-999)",
+             |  "continueOrExit": "exit"
```

Comment:
there's the case where we get `"continue"` where it'd be nice to have a unit test as well in the appropriate place